stable/oauth2-proxy/values.yaml

```diff
@@ -10,6 +10,55 @@ config:
   # Example:
   # existingSecret: secret
   cookieSecret: "XXXXXXXXXX"
+  secretConfig:
+    # If existingSecret is set, all three values are expected to be in
+    # the specified existingSecret. .secretConfig.clientID.keyName,
+    # .secretConfig.clientSecret.keyName and .secretConfig.cookieSecret.keyName will
+    # be respected. All other values in secretConfig are ignored.
+    #
+    # If existingSecret not set:
+    # If .secretConfig.clientID.write == true, clientID will be written to secret
```

Comment:
What is the use case for this (`existingSecret: nil` and `write: true`)?

Reply:
Its needed for the chart to continue to behave as it always has. Backwards compatibility.